src/markitdown/api.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import FastAPI, File, UploadFile, HTTPException
+from fastapi.responses import FileResponse
+from markitdown import MarkItDown
+import os
+
+app = FastAPI()
+
+@app.post("/convert")
+async def convert(file: UploadFile = File(...)):
+    if not file.filename:
+        raise HTTPException(status_code=400, detail="No file uploaded")
+
+    try:
+        contents = await file.read()
+        temp_file_path = f"/tmp/{file.filename}"
```

Comment:
if multiple requests are accepted concurrently with the same filename, they can conflict
(nb: AFAICT this currently does not occur because requests are handled sequentially)